main_js.go

```diff
@@ -907,6 +922,18 @@ func (pe *pcdeditor) runImpl(ctx context.Context) error {
 			}
 			pe.logPrint("pcd file exported")
 			promise.resolved(blob)
+		case promise := <-pe.chExportSelectedPCD:
+			pe.logPrint("exporting selected points as pcd file")
```

Comment:
Does it actually export to file?

Reply:
Exported data (stored to clipboard) is in pcd file format.
User can call `exportSelectedPCD()` API directly and just write to the file instead of storing to the clipboard.